package.json

```diff
@@ -31,9 +31,9 @@
   "scripts": {
     "dev": "concurrently --kill-others \"npm run build:watch\" \"npm run dev:server\"",
     "dev:server": "browser-sync start --config bs-config.json",
-    "prebuild": "rimraf build",
-    "build": "tsc -b ./tsconfig.cjs.json ./tsconfig.esm.json && echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
-    "build:watch": "tsc -b --watch ./tsconfig.cjs.json ./tsconfig.esm.json && echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
+    "prebuild": "shx rm -r ./build && shx mkdir -p ./build/cjs && shx echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
```

Comment:
I have yet to test this under Windows, so let's hold off on merging this PR until it is.

Reply:
Tested, and worked on my windows machine.